.github/workflows/publish-newtonsoft-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Newtonsoft tag created workflow
+
+on:
+  push:
+    tags:
+    - 'newtonsoft-*'
```

Comment:
This makes sure that it runs **only** for when we want to release the Newtonsoft package.